app/controllers/organizations_controller.rb

```diff
@@ -1,15 +1,22 @@
 class OrganizationsController < AdminController
   include Pagination
+  skip_after_action :verify_authorized
 
   def index
     authorize(:dashboard, :show?)
+    current_admin.can?(:view_reports, :facility_group)
```

Comment:
We need an `authorized?` method similar to `can?` that will throw a `Pundit::NotAuthorizedError` equivalent instead of returning a boolean.